python/helpers/requirements.txt

```diff
@@ -6,7 +6,8 @@ hashin==1.0.2; python_version < '3.9'
 hashin==1.0.3; python_version >= '3.9'
 pipenv==2024.0.2
 plette==2.1.0
-poetry==1.8.5
+poetry==1.8.5; python_version < "3.9"
+poetry==2.0.0; python_version >= "3.9"
```

Comment:
This effectively bumps to Poetry 2.0.0... that's a much bigger change than just pinning poetry 1.8.5

I would not be surprised if this part of this change introduces a bunch of breakage that needs fixing...

Reply:
Thanks for the feedback @jeffwidman! I wasn't aware the 3.8 deprecation was so close to being dropped. I agree that there will be breaking changes when bumping to Poetry 2.0, but I unfortunately can't work on fixing those at the moment.